src/sentry/incidents/endpoints/organization_alert_rule_details.py

```diff
@@ -65,6 +72,55 @@ def fetch_alert_rule(request: Request, organization, alert_rule):
     return Response(serialized_rule)
 
 
+def update_alert_rule(request: Request, organization, alert_rule):
+    data = request.data
+    organization_id = data.get("organizationId")
+    if not organization_id:
```

Comment:
We used to be able to grab the organization ID from `project.organization_id`, but the `organization` passed in is just a string, so we need to look it up. In some cases the ID is available in the request data, but I ran into some tests (that I'm assuming reflect real life) that didn't have it and I needed to look it up through projects. I'm being overly cautious here since this is a heavily trafficked endpoint, it's very likely we have it in the request.